packages/theme/stories/introduction.stories.mdx

```diff
@@ -15,35 +15,21 @@ import { CornerBlock } from "docs/components/CornerBlock";
-over the last few years and CSS custom properties are one of the most powerful of the new features. Our theming mechanism
-is built 100% on CSS custom properties. Embracing vanilla CSS offers performance advantages over CSS-in-JS solutions
-(these have improved but most still involve some runtime overhead).
-
-More importantly, with vanilla CSS making up the foundation of a Toolkit theme, the theme itself can be published and
-consumed independently from the component libraries. It can be used in contexts where Toolkit components are not even used,
-e.g. static code deployment in CMS type systems. Separation of styling into CSS files also reduces the tie-in with React,
-which might not be important in the short term, but longer term opens up the possibility that we could share styling assets
-with non-react components.
-
-#### Theme CSS + Component CSS
-
+### Explaining our approach
+
+For the Toolkit, our design system is built simply from vanilla CSS. There are a number of factors for this decision, including the following:
+* Since we don't rely on any third party styling dependencies, we have been completely free in our approach to styling.
+* There has been a lot of innovation in the CSS world over the last few years and CSS custom properties are one of the most powerful of the new features. Our theming mechanism is built 100% on CSS custom properties. 
+* While CSS-in-JS solutions have improved in terms of performance most still involve some runtime overhead, embracing vanilla CSS removes this overhead
+* With vanilla CSS making up the foundation of a Toolkit theme, the theme itself can be published and consumed independently from the component libraries. It can be used in contexts where Toolkit components are not even used, e.g. static code deployment in CMS type systems. 
+* Separation of styling into CSS files also reduces the tie-in with React, which might not be important in the short term, but longer term opens up the possibility that we could share styling assets with non-react components.
 The pattern adopted by the UI Toolkit for styling provides not only the mechanism by which Toolkit components
 themselves are styled, it also provides a consistent approach for styling all elements of your application. Most
 applications will probably contain some mix of Toolkit components, plain HTML infrastructure, often for layout purposes,
 custom components you build yourself, and maybe some third party components. You should be able to make use of the
 styling primitives offered by the Toolkit theme to help style all of these. The goal is to make it as easy as possible to
-achieve consistency of styling across the entire application.
-
-The entire solution is built around vanilla CSS (in other words, this is not a `CSS-in-JS` solution). The CSS comprises
-two kinds of declarations:
+achieve consistency of styling across your entire application.
+The CSS comprises the following two kinds of declarations:
```

Comment:
Is the removal of a new line intentional here?

Reply:
No, fixed now